test/LibraryManager.IntegrationTest/AddCientSideLibrariesFromUITests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.IO;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Web.LibraryManager.IntegrationTest.Services;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class AddCientSideLibrariesFromUITests : VisualStudioLibmanHostTest
+    {
+        private ProjectTestExtension _webProject;
```

Comment:
nit: sort these so consts are together and private fields are together.

Reply:
Updated in iteration 7

---
In reply to: [206379420](https://github.com/aspnet/LibraryManager/pull/332#discussion_r206379420) [](ancestors = 206379420)